src/github.com/getlantern/lantern-ui/app/partials/feed.html

```diff
@@ -1,6 +1,6 @@
 <a class="item" target="_blank" href="{{feed.link}}" ng-click="trackFeed(feedsTitle)">
   <div ng-if="feed.image" class="feed-thumbnail">
-    <img ng-src="{{feed.image}}"/>
+    <img ng-src="{{feed.image}}" on-error="hideImage(feed)"/>
```

Comment:
`on-error` doesn't seem to work in the example at http://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onerror